src/Our.Umbraco.HeadRest/Web/Mvc/HeadRestResult.cs

```diff
@@ -1,15 +1,52 @@
-﻿using Newtonsoft.Json;
+﻿using System;
+using System.IO;
+using System.Text;
+using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json;
 using Newtonsoft.Json.Serialization;
-using Umbraco.Web.Mvc;
 
 namespace Our.Umbraco.HeadRest.Web.Mvc
 {
-    internal class HeadRestResult : JsonNetResult
+    internal class HeadRestResult : ActionResult
```

Comment:
`JsonNetResult` is no longer available, so I copied the code from it and adapted it for netcore to make `HeadRestResult` work the same way and serialize using the serializer settings we want it to.